docs/deploy_graphscope_on_local.md

```diff
@@ -1 +1,193 @@
 # Deploy GraphScope on Local
+
+This guide will walk you through the process of installing and using GraphScope on your local machine.
+
+## Prerequisites
+- Python 3.7 - 3.11
+- Ubuntu 20.04 or later
+- CentOS 7 or later
+- OpenJDK 8 or later (If you want to use GIE)
+- macOS 11.2.1 (Big Sur) or later, with both Intel chip and Apple M1 chip
+
+## Installation
+GraphScope is distributed by [Python wheels](https://pypi.org/project/graphscope) , and could be installed by [pip](https://pip.pypa.io/en/stable/) directly.
+
+- Use pip to install latest stable graphscope package
+
+```bash
+python3 -m pip install graphscope --upgrade
+```
+
+- Use Aliyun mirror to accelerate downloading if you are in CN
+
+```bash
+python3 -m pip install graphscope -i http://mirrors.aliyun.com/pypi/simple/ --trusted-host=mirrors.aliyun.com  --upgrade
+```
+
+
+
+The above command will download all the components required for running GraphScope in standalone mode to your local machine.
+
+Alternatively, you can use a  Docker development environment to get started
+
+```bash
+docker run --name dev -it --shm-size=4096m ubuntu:latest
+
+# inside the docker
+apt-get update -y && apt-get install python3-pip default-jdk -y
+python3 -m pip install graphscope ipython tensorflow
+```
+
+## Launch graphscope
```

Comment:
- Launch GraphScope standalone in Python
- Launch a GraphScope cluster on local with minikube
- No need to repeat the demo about run graph tasks, give link to tutorials.